packages/components/src/theme/theme-dark/theme-dark-components.css

```diff
@@ -1,13 +1,55 @@
+/* stylelint-disable custom-property-empty-line-before */
 /* stylelint-disable alpha-value-notation */
 :root {
-  --dh-color-loading-spinner-primary: var(--dh-color-accent-default-bg);
+  --dh-color-loading-spinner-primary: var(--dh-color-accent-bg);
   --dh-color-loading-spinner-secondary: hsla(var(--dh-color-gray-800-hsl), 0.5);
   --dh-color-quickactions-bg: hsla(var(--dh-color-visual-gray-hsl), 0.9);
   --dh-color-radial-reaction: hsla(var(--dh-color-visual-gray-hsl), 0.6);
   --dh-color-colorpicker-border: hsla(var(--dh-color-visual-gray-hsl), 0.1);
 
+  /* Login */
+  --dh-color-login-form-bg: var(--dh-color-gray-400);
+  --dh-color-login-status-message: var(--dh-color-gray-600);
+
+  /* We should base this color on something from the palette, but for now just
+   * hard coding the original value */
+  --dh-color-login-animation-fg: #343e5d;
+  --dh-color-login-animation-bg: var(--dh-color-gray-75);
+  --dh-color-login-animation-grid: var(--dh-color-gray-300);
```

Comment:
Yeah I wonder what values we should actually be using here, unclear to me why this doesn't fit into our palette @dsmmcken 
Side note, I don't particularly like the name `login-animation` - yes, we use that animation only on the login screen, but there's nothing about the animation/component itself that is inherently tied to login; the animation could be used anywhere.

Reply:
Needs to be composed with alpha now, and remove the animation from computing it's own alpha on primary-dark with a new constant.

Adjust the animation to have something like:
```js
// obviously, these need to be dynamically replaced with whatever cssvar -> js variable thing we do.
FOREGROUND_STROKE = hsla(    var(--dh-color-accent-hsl),    0.2  ); // something that looks like primary dark
FOREGROUND_FILL = hsla(    var(--dh-color-accent-hsl),    0.08  ); // new variable, with even more opacity
```
and the `FILL_OPACITY: 0.25` variable can be deleted.